benchutils/tests/data/expected_genus_profile.txt

```diff
@@ -0,0 +1,9 @@
+@SampleID:
+@Version:0.9.1
+@Ranks:superkingdom|phylum|class|order|family|genus|species|strain
+
+@@TAXID	RANK	TAXPATH	TAXPATHSN	PERCENTAGE	_CAMI_genomeID	_CAMI_OTU
+1	genus	9|8|7|6|5|1	Bacteria|blah8|blah7|blah6|blah5|blah1	50
```

Comment:
I feel the current design is highly dependent on the CAMI nomenclature. I'm not saying that it's not okay. Just feel that it disables some future flexibility.

Reply:
I think you're right. Future issue to make it more general.